res_users_password_security/models/res_users_pass_history.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from openerp import fields, models
+
+
+class ResUsersPassHistory(models.Model):
+    _name = 'res.users.pass.history'
+    _description = 'Res Users Password History'
+
+    _order = 'user_id, date desc'
+
+    user_id = fields.Many2one(
+        string='User',
+        comodel_name='res.users',
+        index=True,
```

Comment:
If we delete a user: Should we store the pass history?
If the answer is `no`: you could use `ondelete=cascade`